README.md

```diff
@@ -1,22 +1,195 @@
+| `/circleci insight workflows` | Get summary of metrics for workflows over past 90 days.                 |
+| `/circleci insight jobs`      | Get summary of metrics for jobs over past 90 days for a given workflow. |
+
+#### Manage CircleCI projects
+
+|                                        |                                 |
+| -------------------------------------- | ------------------------------- |
+| `/circleci project list-followed`      | List followed projects.         |
+| `/circleci project recent-build`       | List the 10 last builds.        |
+| `/circleci project env list`           | List environment variables.     |
+| `/circleci project env add name value` | Add an environment variable.    |
+| `/circleci project env remove name`    | Remove an environment variable. |
+
+#### Manage pipelines
+
+**Note:** Use the `all`, `recent`, and `mine` subcommands get the pipelineID for a particular pipeline.
+
+|                                     |                                                                                                      |
+| ----------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `/circleci pipeline trigger branch` | Trigger pipeline for a given branch.                                                                 |
+| `/circleci pipeline trigger tag`    | Trigger pipeline for a given tag.                                                                    |
+| `/circleci pipeline workflows`      | Get list of workflows for given pipeline.                                                            |
+| `/circleci pipeline recent`         | Get list of all recently ran pipelines.                                                              |
+| `/circleci pipeline all`            | Get list of all pipelines for a project.                                                             |
+| `/circleci pipeline mine`           | Get list of all pipelines triggered by you for a project.                                            |
+| `/circleci pipeline get`            | Get information about a single pipeline given pipeline ID.                                           |
+| `/circleci pipeline get`            | Get information about a single pipeline for a given project by passing the unique pipelineID number. |
+
+#### Manage workflows
+
+**Note:** Use the `/circleci pipeline workflows` command to get the workflowID.
+
+|                             |                                     |
+| --------------------------- | ----------------------------------- |
+| `/circleci workflow get`    | Get information about a workflow.   |
+| `/circleci workflow jobs`   | Get jobs list for a given workflow. |
+| `/circleci workflow rerun`  | Rerun a given workflow.             |
+| `/circleci workflow cancel` | Cancel a given workflow.            |
+
+### Webhooks Notifications
```

Comment:
```suggestion
### Webhooks notifications
```
Should notifications be lower case?

Reply:
Case across these docs are inconsistent. Once I've migrated all of the content over, I'll do a quick consistency sweep.